src/components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './ProductCard.scss';
+import { Link } from 'react-router-dom';
+
+const ProductCard = ({ src, title, price, rating, reviewCount, content }) => {
+  const priceName = price.toLocaleString('ko-KR');
+  return (
+    <div className="productCard">
+      <Link to="/" className="productLink">
+        <div className="basic" />
+        <img src={src} alt="제품이미지" className="productImg" />
```

Comment:
alt 값에 어떤 제품인지 알 수 있게 title로 부여해도 괜찮을 것 같네요!
```suggestion
        <img src={src} alt={title} className="productImg" />
```